korman/properties/modifiers/avatar.py

```diff
@@ -78,6 +78,16 @@ def requires_actor(self):
         return True
 
 
+# Use xSitCam.py for when we want a camera to pop up
+sitting_pfms = {
```

Comment:
```suggestion
sitting_pfm = {
```

In conjunction with the other suggestion.